.github/scripts/findUnusedStyles.ts

```diff
@@ -0,0 +1,561 @@
+import * as fs from 'fs';
+import {globSync} from 'glob';
+import * as path from 'path';
+import * as ts from 'typescript';
+import TSCompilerUtils from '../../scripts/utils/TSCompilerUtils';
+
+type StyleDefinition = {
+    key: string;
+    file: string;
+    line: number;
+    column: number;
+};
+
+// Static patterns and constants - created once
+const STYLE_FILE_EXTENSIONS = '**/*.{ts,tsx,js,jsx}';
+const STYLE_KEY_SKIP_PATTERNS = ['default', 'exports', 'module', 'require', 'import', 'from', 'theme', 'colors', 'variables', 'CONST', 'Platform', 'StyleSheet'];
+
+class ComprehensiveStylesFinder {
+    private rootDir: string;
+
+    private styleDefinitions = new Map<string, StyleDefinition>();
+
+    private fileContents = new Map<string, string>();
+
+    constructor(rootDir: string) {
+        this.rootDir = rootDir;
+    }
+
+    findUnusedStyles(): StyleDefinition[] {
```

Comment:
NAB but I like to include the `public` keyword even though it's not necessary:

```suggestion
    public findUnusedStyles(): StyleDefinition[] {
```